app/views/helpers/calcBreakdownHelper.scala.html

```diff
@@ -59,8 +59,8 @@
             </tr>
         }
 
-        @if(breakdown.savingsAllowanceSummaryData == 0 && breakdown.additionalAllowances == 0) {
-        <tr id="personal-allowance-section">
+        @if(breakdown.savingsAllowanceSummaryData == 0 && breakdown.additionalAllowances == 0 && breakdown.giftAid.paymentsMade == 0) {
+        <tr id="personal-allowance-section" class="no-border-bottom">
```

Comment:
Remove this one class, if savings allowance, additional allowance and giftaid is missing, personal allowance should have that border :+1: